README.md

```diff
@@ -1,12 +1,17 @@
 # movegroup_interface_demo
 A package designed to help learning ROS MoveIt Move Group C++ Interface.
 
-Although this package should be usable with any properly setup MoveIt configuration package, the development and testing has been conducted using xArm7 ROS support available at https://github.com/xArm-Developer/xarm_ros
+The underlying idea for this package is to provide easy-to-understand coding examples to illustrate the capabilities of [Move Group C++ Interface](http://docs.ros.org/kinetic/api/moveit_ros_planning_interface/html/classmoveit_1_1planning__interface_1_1MoveGroupInterface.html "moveit::planning_interface::MoveGroupInterface Class Reference"). For that reason:
++ the coding aims to be as minimalistic as possible with intuitive variable names;
++ different features are exemplified individually in separate ROS nodes to make code readable for beginners in computer programming.
+
+Although this ROS package should be usable with any properly set up MoveIt configuration package, the development and testing has been conducted using xArm7 ROS support available at https://github.com/xArm-Developer/xarm_ros
 
```

Comment:
Since this is for the very beginners, I think the following might be helpful.

```suggestion
## Prerequisites
1. If you don't have the movegroup interface package yet, you can easily install it with the following command.
```bash
sudo apt install ros-$ROS_DISTRO-moveit-ros-planning-interface
```
2. You also need a MoveIt configuration package to use this demo. 
Although this ROS package should be usable with any properly set up MoveIt configuration package, the development and testing has been conducted using xArm7 ROS support available at https://github.com/xArm-Developer/xarm_ros
```

Reply:
done. see 2777eff